service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/PartyAddedToCaseContentProvider.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences;
+import uk.gov.hmcts.reform.fpl.service.DateFormatterService;
+import uk.gov.hmcts.reform.fpl.service.HearingBookingService;
+
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.PARTY_ADDED_TO_CASE_BY_EMAIL_NOTIFICATION_TEMPLATE;
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.PARTY_ADDED_TO_CASE_THROUGH_DIGITAL_SERVICE_NOTIFICATION_TEMPLATE;
+import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.EMAIL;
+
+@Service
+public class PartyAddedToCaseContentProvider extends AbstractEmailContentProvider {
+
+    private final PartyAddedToCaseByEmailContentProvider partyAddedToCaseEmailContentProvider;
+    private final PartyAddedToCaseThroughDigitalServiceContentProvider
+        partyAddedToCaseThroughDigitalServicelContentProvider;
+
+    @Autowired
+    public PartyAddedToCaseContentProvider(@Value("${ccd.ui.base.url}") String uiBaseUrl,
+                                           DateFormatterService dateFormatterService,
+                                           HearingBookingService hearingBookingService,
+                                           PartyAddedToCaseByEmailContentProvider partyAddedToCaseEmailContentProvider,
+                                           PartyAddedToCaseThroughDigitalServiceContentProvider
+                                                   partyAddedToCaseThroughDigitalServicelContentProvider) {
+        super(uiBaseUrl, dateFormatterService, hearingBookingService);
+        this.partyAddedToCaseEmailContentProvider = partyAddedToCaseEmailContentProvider;
+        this.partyAddedToCaseThroughDigitalServicelContentProvider
+            = partyAddedToCaseThroughDigitalServicelContentProvider;
+    }
+
+    public Map<String, Object> getPartyAddedToCaseNotificationParameters(CaseDetails caseDetails,
+                                        RepresentativeServingPreferences servingPreferences) {
+        if (servingPreferences.equals(EMAIL)) {
```

Comment:
When using `.equals()` it is better to do `CONSTANT.equals(variable)` for null safety, however with enums you just need to do `==` as there will ever only be one instance of a enum

Reply:
cool :)